src/utils/script_args.py

```diff
@@ -32,18 +32,16 @@ def generic_script_init(description: str) -> ScriptArgs:
     )
     parser.add_argument(
         "--post-tx",
-        type=bool,
+        action="store_true",
         help="Flag indicating whether multisend should be posted to safe "
         "(requires valid env var `PROPOSER_PK`)",
-        default=False,
     )
     parser.add_argument(
         "--dry-run",
-        type=bool,
+        action="store_true",
```

Comment:
Could you also fix the comment 2 lines below, i.e., L43?

Reply:
Fixed in https://github.com/cowprotocol/solver-rewards/pull/422/commits/73143b74d2b2202a141fc670eacc1299221e44c9